test/extended/cli/mustgather.go

```diff
@@ -146,6 +146,19 @@ var _ = g.Describe("[sig-cli] oc adm must-gather", func() {
 					return nil
 				}
 
+				// TODO: remove when https://github.com/openshift/must-gather/pull/162 has merged
+				if !strings.HasSuffix(path, ".terminating.gz") {
+					return nil
+				}
+
+				if !strings.Contains(path, "-termination-") && !strings.HasSuffix(path, "termination.log.gz") {
+					// these are expected, but have unstructured log format
+					return nil
+				}
+
+				isAuditFile := strings.Contains(path, "-audit-") || strings.HasSuffix(path, "audit.log.gz")
```

Comment:
at this point, we expect only `audit` compliant files.

Reply:
added comment